### DISCUSSION TITLE
How do I get crazyflie to take off once I've burned the TinyMPC code into crazyflie?

### DISCUSSION THREAD

**Discussion by whY-note** [TinyMPC/TinyMPC#45] [Q&A] [answered]
How do I get crazyflie to take off once I've burned the TinyMPC code into crazyflie?

There's nothing wrong with the hardware of my crazyflie. But after I  burned the TinyMPC code into crazyflie, I do not know how to make crazyflie  take off, could you help me to solve this problem?

Your prompt reply will be highly appreciated.

Thank you!

**Comment by khairesearch** [comment, marked as answer]
Hi,

Please take a look at https://github.com/A2R-Lab/tinympc-crazyflie/tree/main
